game/squadrons/airwing.py

```diff
@@ -37,9 +37,10 @@ def squadron_def_for_aircraft(self, aircraft: AircraftType) -> SquadronDef:
         return self.squadron_def_generator.generate_for_aircraft(aircraft)
 
     def unclaim_squadron_def(self, squadron: Squadron) -> None:
-        for squadron_def in self.squadron_defs[squadron.aircraft]:
-            if squadron_def.claimed and squadron_def.name == squadron.name:
-                squadron_def.claimed = False
+        if squadron.aircraft in self.squadron_defs:
+            for squadron_def in self.squadron_defs[squadron.aircraft]:
+                if squadron_def.claimed and squadron_def.name == squadron.name:
+                    squadron_def.claimed = False
```

Comment:
the downside of this unclaim is that it just checks for the name which could be changed by the user. The only way to handle this would be to save the original def in the squadron during generation. As the only point where this could happen is during the new game wizard and only for the special condition that the name was changed i personally think this is acceptable

Reply:
I think I agree that this is fine (but I haven't read the full change here so I don't have all the context.

An alternative that I think we'll need to pursue at some point for a handful of types: add a UUID field.